### PR TITLE
Modify README.md for Windows build

### PR DESCRIPTION
**Pull request by gcjjyy**
To build on Windows,
Need git config before git clone.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [19bcfd14664193137f7790b6b91f2140b01dd7e9] - Code Rule Check OK.

File: README.md [hunk 1]
```diff
@@ -45,6 +45,11 @@ When you become familiar to the TizenRT build system, you may use the *dbuild.sh
 As the build script is running based on Docker, it requires *sudo* for root permission.  
 To run Docker without *sudo*, refer to https://docs.docker.com/install/linux/linux-postinstall/.
 
+If your OS is Windows, you need to configure git like below before git clone:
+```
```
Comment (juitem) [inline README.md]: For windows environment, configuring CRLF is required as below.

Comment (gcjjyy) [reply README.md]: Fixed!

**Comment by seinfra** [comment]
Target : [6eefddd4700ebbfb0545db0cd77672e33fd56dbf] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6eefddd4700ebbfb0545db0cd77672e33fd56dbf] - Code Rule Check OK.

File: README.md [hunk 2]
```diff
@@ -45,6 +45,11 @@ When you become familiar to the TizenRT build system, you may use the *dbuild.sh
 As the build script is running based on Docker, it requires *sudo* for root permission.  
 To run Docker without *sudo*, refer to https://docs.docker.com/install/linux/linux-postinstall/.
 
+For windows environment, configuring CRLF is required as below.
```
Comment (sunghan-chang) [inline README.md]: I think this is not a build step. Let's move this inside `Environment setup`, next of `2. Getting TizenRT source code`.

File: README.md [hunk 3]
```diff
@@ -21,6 +21,11 @@ Please refer to [Manual Setup Build Environment](docs/HowToSetEnv.md).
 
 For more information of libraries in the TizenRT Docker Image, see https://hub.docker.com/r/tizenrt/tizenrt/.
 
+Especially For Windows environment, configuring CRLF is required as below.
```
Comment (sunghan-chang) [inline README.md]: As I left a comment before, let's move this next of `2. Getting TizenRT source code`. Because this should be executed on forked folder.

**Comment by seinfra** [comment]
Target : [8e8dac67511ffbb2060850b1f29e750821494edb] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [8e8dac67511ffbb2060850b1f29e750821494edb] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [8a57ef12297fb0048862a754b603628058be546c] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [8a57ef12297fb0048862a754b603628058be546c] - Code Rule Check (C++) OK.

**Comment by sunghan-chang** [comment]
@gcjjyy As we discussed on offline, we had better guide the project option instead of global.

**Comment by sunghan-chang** [comment]
@gcjjyy Please find a link which provides by GiyHub as shown below:
https://help.github.com/en/articles/setting-your-username-in-git

In local, if we input a command without `--global`, it looks setting of local only.
So, yours can be as shown below:
```
git config core.autocrlf input
```

Could you test that?

**Pull request merged**